handel/src/services/route53zone/index.ts

```diff
@@ -75,21 +75,21 @@ function getCompiledRoute53Template(ownServiceContext: ServiceContext<Route53Zon
  */
 
 export function check(serviceContext: ServiceContext<Route53ZoneServiceConfig>, dependenciesServiceContexts: Array<ServiceContext<ServiceConfig>>): string[] {
-    const errors = [];
+    const errors: string[] = checkPhase.checkJsonSchema(`${__dirname}/params-schema.json`, serviceContext);
 
     const params = serviceContext.params;
 
     if (!params.name) {
```

Comment:
This first if branch can be removed because it's now checked in the JSON schema.